Project: npm run prettify fails on Windows

# Bug Report

## Prerequisites

- [x] Can you reproduce the problem in a [MWE](https://en.wikipedia.org/wiki/Minimal_working_example)?
- [x] Are you running the latest version?
- [x] Did you perform a search in the issues?

## Environment Details and Version

Windows 10, a fresh clone of the Piral repository,

## Description

Running `npm run prettify` on Windows currently fails with the following output:
```
npm run prettify

> piral@ prettify C:\Development\Smapiot\Piral
> prettier --config prettier.config.js --write 'src/**/*.{ts,tsx}'

[error] No matching files. Patterns tried: 'src/**/*.{ts,tsx}' !**/node_modules/** !./node_modules/** !**/.{git,svn,hg}/** !./.{git,svn,hg}/**
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! piral@ prettify: `prettier --config prettier.config.js --write 'src/**/*.{ts,tsx}'`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the piral@ prettify script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Manuel\AppData\Roaming\npm-cache\_logs\2019-12-03T14_03_57_513Z-debug.log
```

## Steps to Reproduce

Run `npm run prettify` on Windows.

## Expected behavior

Prettier executes normally.

## Actual behavior

Prettier is not executed.

## Possible Origin / Solution

This can simply be fixed by replacing single quotes `'` with double quotes `"` in `package.json`:

```
"prettify": "prettier --config prettier.config.js --write \"src/**/*.{ts,tsx}\""
```

## Discussion

Can someone with another OS verify that the suggested change does not break the script on non Windows machines?

--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
     "watch:demo-full": "piral debug src/samples/sample-piral/src/index.html",
     "build:demo-full": "piral build src/samples/sample-piral/src/index.html --target dist/demo-full/index.html",
     "lint": "tslint -c tslint.json 'src/**/*.{ts,tsx}' --project tsconfig.json",
-    "prettify": "prettier --config prettier.config.js --write 'src/**/*.{ts,tsx}'",
+    "prettify": "prettier --config prettier.config.js --write \"src/**/*.{ts,tsx}\"",
     "test": "jest --forceExit",
     "prepublish": "yarn build",
     "release": "lerna publish",

